lms/templates/provider/authorize.html

```diff
@@ -1,59 +1,66 @@
-{% extends "main_django.html" %}
+## mako
 
-{% load scope i18n %}
+<%!
+from django.utils.translation import ugettext as _
+from provider.templatetags.scope import scopes
+from django.core.urlresolvers import reverse
+%>
 
-{% block bodyclass %}oauth2{% endblock %}
+<%inherit file="../main.html"/>
 
-{% block body %}
+<%block name="bodyclass">oauth2</%block>
+
+<%block name="body">
 <div class="authorization-confirmation">
-  {% if not error %}
+  % if not error:
   <p>
-  {% blocktrans with application_name=client.name %}
-    <strong>{{ application_name }}</strong> would like to access your data with the following permissions:
-  {% endblocktrans %}
+  ${_(
```

Comment:
I forget if I've shown you this, but you should follow the safe templating documentation here:

http://edx.readthedocs.org/projects/edx-developer-guide/en/latest/conventions/safe_templates.html